br/pkg/summary/collector.go

```diff
@@ -225,9 +232,49 @@ func (tc *logCollector) Summary(name string) {
 		logFields = append(logFields, zap.Uint64(logKeyFor(name), data))
 	}
 
+	return logFields
+}
+
+func (tc *logCollector) Summary(name string) {
+	logFields := tc.summaryDetail(true)
 	tc.log(name+" success summary", logFields...)
 }
 
+func field2String(field zapcore.Field) string {
+	switch field.Type {
+	case zapcore.StringType:
+		return field.String
+	case zapcore.Int64Type, zapcore.Int32Type, zapcore.Uint32Type, zapcore.Uint64Type:
+		return fmt.Sprintf("%v", field.Integer)
+	case zapcore.Float64Type:
+		return fmt.Sprintf("%v", math.Float64frombits(uint64(field.Integer)))
+	case zapcore.StringerType:
+		return field.Interface.(fmt.Stringer).String()
+	case zapcore.DurationType:
+		return ((time.Duration)(field.Integer)).String()
+	default:
+		return ""
+	}
+}
+
+func (tc *logCollector) CollectToStr(cmdName string) string {
```

Comment:
It there some reason we must use the log-like format here? Considering that our users usually won't read the backup meta directly, the format could be more friendly to computers. I would prefer to write them as a more structural format, say, JSON?
(Another benefit of using json is that we can use the zap itself to encode it)